index.html

```diff
@@ -196,40 +207,54 @@ <h2>About</h2>
 <!-- foooter -->
 <footer class="contact_section">
     <div class="mobile_logo"></div>
+
+    <div class="footer_div">
+
+
     
-      <h4 class="title-contact text_header_font">Get Started</h4>
-
-      <p class="section-desc text_body_font">
-        Hello!I am a software developer! I can 
-        help you build a product, feature or website.
-        Take a look of my works. If you like what you see 
-        and have a project you need coded, don't hesitate 
-        and contact me.
-      </p>
-      <form action="https://formspree.io/f/mzbovllg" method="post">
-          <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
-          <input type="email" name="Email" placeholder="Enter Your Email" required>
-          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
-          <button class="" type="submit" name="submit-btn">Start collaboration</button>
-      </form>
-
-      <div class="bottom_row">
-        <a class="text-button text_body_font resume_color">
-            <p>Get my Resume</p>
-            <div class="arrow_upload"></div>
-        </a>
-
-        <div class="divicon">
-            <ul class="sociallist">
-           <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/linkdin.png" alt="linkdin"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/medium.png" alt="medium"></a></li>
-           <li class="sociallist_link"><a href="#"><img src="img/github.png" alt="github"></a></li>
-           
-            </ul>
+    
+        <div class="contact_div">
+
+    
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can 
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see 
+                and have a project you need coded, don't hesitate 
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+                <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
+                <input type="email" name="Email" placeholder="Enter Your Email" required>
+                <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
+                <button class="" type="submit" name="submit-btn">Start collaboration</button>
+            </form>
         </div>
 
-      </div>
+                <div class="bottom_row">

```

Comment:
- [ ] The content spacing and alignment in the footer should match the original website.
- Your website  
![Screenshot from 2022-03-27 03-20-49](https://user-images.githubusercontent.com/26767383/160258555-db6d5090-8b53-4b27-a36a-b7777326e60e.png)

- Figma website 
![Screenshot from 2022-03-27 03-20-55](https://user-images.githubusercontent.com/26767383/160258594-13dbd50d-6bce-44a2-a3cf-3c3108b48bbc.png)